src/EditorFeatures/TestUtilities/Workspaces/EditorTestWorkspace.cs

```diff
@@ -31,44 +31,26 @@
 
 namespace Microsoft.CodeAnalysis.Test.Utilities;
 
-public partial class EditorTestWorkspace : TestWorkspace<EditorTestHostDocument, EditorTestHostProject, EditorTestHostSolution>, ILspWorkspace
+public partial class EditorTestWorkspace : TestWorkspace<EditorTestHostDocument, EditorTestHostProject, EditorTestHostSolution>
```

Comment:
we're not using this anymore for any LSP tests that require mutation, so no need for it to implement the LSP mutation methods.  Only used by doc outline tests at the moment (which work with any workspace).